### ISSUE TITLE
Geometries created within query are not rendered

### ISSUE THREAD

**Issue by magdasalatka** [ad-freiburg/qlever-petrimaps#23]
It is impossible to visualize geometries created within the query:

```sparql
PREFIX geo: <http://www.opengis.net/ont/geosparql#>

SELECT *
WHERE {

  BIND(STRDT("POINT(8 47)", geo:wktLiteral) AS ?point)
}
```

**Comment by patrickbr** [comment]
Yes, good catch. The reason is the way petrimaps works: it first loads all the geometries stored in QlLver during startup. Aftwards, only the QLever-internal IDs are transferred to petrimaps, and the corresponding geometry is looked up by petrimaps.

Since the geometry is generated on-the-fly in your query, petrimaps does not have it cached. We are currently experimenting with ways to show the result of arbitrary SQL queries to a PostGIS database in petrimaps. I expect that in the wake of this feature, we will also add the ability to display SPARQL results containing non-cached geometries.
